deploy/ArbitrumDeploy.js

```diff
@@ -55,9 +55,42 @@ module.exports = async (hre) => {
         from: deployer,
         log: true,
         contract: "ChainlinkOracleWrapper",
-        args: [KovanEurUsdOracle.address],
+        args: [KovanEurUsdOracle.address, deployer],
     })
 
+    // deploy SMA PriceObserver
+    const priceObserver = await deploy("EthUsdPriceObserver", {
+        from: deployer,
+        log: true,
+        contract: "PriceObserver",
+    })
+
+    // deploy SMA Oracle
+    const smaOracleWrapper = await deploy("EthUsdPriceSMAOracle", {
+        from: deployer,
+        log: true,
+        contract: "SMAOracle",
+        args: [
+            RinkebyEthUsdOracle.address,
```

Comment:
The network here differs from the one on line 58 (i.e., Rinkeby versus Kovan, respectively). Is this intentional?

Reply:
a little, i was mainly focused on making the logic correct and then maybe make another pr after i deploy to use correct constants for addresses. to my knowledge we deploying on arbitrum rinkeby.

Reply:
I looked at the rinkeby oracles and they are eth rinkeby not arbitrum rinkeby. Will need to find the rinkeby oracles on aribitrum.

Reply:
alright updated with arbitrum rinkeby oracle addresses